README.md

```diff
@@ -112,10 +112,34 @@ Currently the controller does not support creating the PSQL server on its own, m
   oc create -f config/creating_env/postgress_secret.yaml
   ```
 
+## External VM using CNV
```

Comment:
External VM using CNV -> External VM *Worker* using CNV

Reply:
I think it's an assumption that we have CNV - but we didn't add it as a prerequisite

Reply:
Regarding the second comment. CNV is a requirement for local workers. For the external worker, this in only a sample. In practice, there is no point in using CNV based VMs as external workers since one can achieve the same with internal workers

Reply:
I meant - CNV operator is required for the workers that created by the osbuild operator

Reply:
We can add a line about it in the `Requirements` section. But, it how does it relate to this PR?

Reply:
Not specific related but I think it can be part of that PR

Reply:
Still think this should be done separately, but done